docs/http/http-metrics.md

```diff
@@ -143,6 +144,8 @@ SHOULD include the [application root](/docs/http/http-spans.md#http-server-defin
 > Since this attribute is based on HTTP headers, opting in to it may allow an attacker
 > to trigger cardinality limits, degrading the usefulness of the metric.
 
+**[10] `user_agent.synthetic.type`:** This flag can primarily be determined by the contents of the `user_agent.original` attribute. Components that populate the attribute should determine what they consider synthetic or bot traffic, and set this attribute accordingly. This attribute can either be set for self-identification purposes, or on spans detected to be generated as a result of a synthetic request. This attribute is useful for distinguishing between genuine client traffic and synthetic traffic generated by bots or tests.
```

Comment:
nit: we should not mention spans on attribute definition since it's used on different signals